### ISSUE TITLE
acknowledge scope of data in search help message

### ISSUE THREAD

**Issue by JannTibetan** [BuddhistDigitalResourceCenter/BDRC-Lib-App#94]
I searched on every number between P3450-P3459 and only 2 records came up on the app (P3456 and P3458). On the website there are person records for all of these numbers.

**Comment by eroux** [comment]
This is a feature in the data, we only import authors in the app, not all persons. The App doesn't have as much data as the website (no places, no work/instance, no non-authors, no copyrighted or restricted works, etc.), by design... I think it's a little too late in the process to change that but we can think of it for the next release... would that be ok @JannTibetan ?

**Comment by JannTibetan** [comment]
This is perfectly fine with me. I understand why you made this choice. I think we can make the text on the landing page a little more explicit about the fact that this app gives access to the full set of bibliographic data (texts and authors) but, for reasons of app size and UX, does not include much of the rest of the BDRC database.

**Comment by eroux** [comment]
ah yes, good idea! The current text says

```
Search by title such as བཀའ་འགྱུར།, authors by name such as བུ་སྟོན།, texts by title such as  བཀྲ་ཤིས་ཆེན་པོའི་མདོ།, or BDRC RIDs such as MW22084. Unicode Tibetan must be used, except in the case of an RID.

在上方输入检索关键词：书名，如བཀའ་འགྱུར；作者，如བུ་སྟོན།；文本名，如བཀྲ་ཤིས་ཆེན་པོའི་མདོ།；BDRC RID编号如MW22084。除了BDRC RID编号外，输入的关键词必须是 Unicode编码的藏文。

(no translation in Tibetan for some reason)
```

what would be a good wording to acknowledge the limited scope of the data?

**Comment by eroux** [comment]
@JannTibetan what about:

```
Search by title such as བཀའ་འགྱུར།, authors by name such as བུ་སྟོན།, texts by title such as  བཀྲ་ཤིས་ཆེན་པོའི་མདོ།, or BDRC RIDs such as MW22084. Unicode Tibetan must be used, except in the case of an RID. Only open access texts and authors (not all persons) are discoverable in the app.
```

?

**Comment by JannTibetan** [comment]
Do we need to differentiate between Title and Text? What is the difference, actually?

**Comment by eroux** [comment]
I think the wording still uses the tbrc.org vocabulary of **work** (here implicit in `Search [work] by title`) vs **text** (in `texts by title`), but yes, this distinction is less strong on BUDA so perhaps

```
Search text titles such as བཀའ་འགྱུར།, author names such as བུ་སྟོན།, or BDRC IDs such as MW22084. Unicode Tibetan must be used (transliteration is not supported). Only open access texts and only authors (not all persons) are discoverable in the app.
```

**Comment by JannTibetan** [comment]
I added an opening sentence (but it might be too wordy)
The BDRC Lib app provides access to the images and metadata for Tibetan texts in the BDRC archive and the records of their authors. Search text titles such as བཀའ་འགྱུར།, author names such as བུ་སྟོན།, or BDRC IDs such as MW22084. Unicode Tibetan must be used (transliteration is not supported). Only open access texts and only authors (not all persons) are discoverable in the app.

**Comment by eroux** [comment]
looks good to me, thanks!

**Comment by JannTibetan** [comment]
*Tibetan script must be used -- all iPhone keyboards are unicode so let's specify Tibetan script (I think more people know what that means anyway)

**Comment by ngawangtrinley** [comment]
During the last iteration we tested the app with users and the majority agreed on the fact they prefer not having that information. What they could understand without explanations from that text blob was that they should type text names or author names. That is displayed in the search bar. A case of localization vs translation.

![](https://user-images.githubusercontent.com/17675331/122240880-afb2ba00-cef4-11eb-8a6e-3d7f253da7e5.png)
